packages/mobile/src/components/host-remix-contest-drawer/HostRemixContestDrawer.tsx

```diff
@@ -65,7 +65,12 @@ export const HostRemixContestDrawer = () => {
   const [endDateError, setEndDateError] = useState<boolean>(false)
 
   const handleChange = useCallback((date: string, time: string) => {
-    const newDate = mergeDateTime(date, time)
+    if (!date && !time) {
+      setEndDate(null)
+      return
+    }
+
+    const newDate = mergeDateTime(date || time, time || date)
```

Comment:
explain why it's chill to do date || time, time || date?

Reply:
both are a full date string so if either are passed it will have a clean default for the other. this avoids one being null and causing an issue for the mergeDateTime function